docker/hub/Dockerfile

```diff
@@ -35,10 +35,9 @@ RUN apt-get update && \
 # show backtraces
  RUST_BACKTRACE=1 && \
 # build parity
-cd /build&&git clone https://github.com/paritytech/parity && \
+cd /build&&git clone https://github.com/slockit/parity -b in3 && \
```

Comment:
Please undo the changes to the Dockerfile, otherwise you break our release scripts.